docs/docs/usage/version-numbers.md

```diff
@@ -1,17 +0,0 @@
----
```

Comment:
We have several links to this page on the dotty website (e.g. release blog posts).

Reply:
Can we make a redirect from that url to the new one?